src/components/Calculator.js

```diff
@@ -1,6 +1,8 @@
 import React, { useState } from 'react';
```

Comment:
- [ ] Your commit messages are still not very professional. Please take some time to read through [this article](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) and learn to write professional commit messages and best practices.